napari/view_layers.py

```diff
@@ -117,8 +117,10 @@ def _merge_layer_viewer_sigs_docs(func):
 
 def _make_viewer_then(
     add_method: str,
-    args,
-    kwargs,
+    /,
```

Comment:
how args will work after `/`?

Reply:
This is `/` , which affects what comes before it, unlike `*`. Which affects what comes after!

Reply:
ok, i see

Reply:
For future reference:

https://peps.python.org/pep-0570/